x-pack/test/functional/page_objects/accountsetting_page.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+//import { map as mapAsync } from 'bluebird';
+import expect from 'expect.js';
+
+export function AccountSettingProvider({ getService }) {
+  const testSubjects = getService('testSubjects');
+
+  class AccountSettingsPage {
+    async verifyAccountSettings(expectedEmail, expectedUserName) {
+      await testSubjects.click('loggedInUser');
+      const usernameField = await testSubjects.find('usernameField');
+      const userName = await usernameField.getVisibleText();
+      expect(userName).to.be(expectedUserName);
+      const emailIdField = await testSubjects.find('emailIdField');
+      const emailField = await emailIdField.getVisibleText();
+      expect(emailField).to.be(expectedEmail);
+    }
+
+    async changePasswordLink(currentPassword, newPassword) {
```

Comment:
Just a little nit here. Instead of `changePasswordLink`, we should probably change the name to `changePassword`. The link is the `data-test-subj`.